6-Final-Proj/common/types.ts

```diff
@@ -0,0 +1,8 @@
+export type guid = string;
+export type UserID = string;
```

Comment:
not crucial but for better readability i would do
```suggestion
export type UserID = guid;
```